miniapps/meshing/mesh-optimizer.cpp

```diff
@@ -82,7 +82,12 @@
 //   * mesh-optimizer -m ../../../mfem_data/ball-pert.mesh -o 4 -rs 0 -mid 303 -tid 1 -ni 20 -ls 2 -li 500 -fix-bnd
 //   2D non-conforming shape and equal size:
 //     mesh-optimizer -m ./amr-quad-q2.mesh -o 2 -rs 1 -mid 9 -tid 2 -ni 200 -ls 2 -li 100 -bnd -qt 1 -qo 8
-
+//
+//   2D untangling:
+//     mesh-optimizer -m jagged.mesh -o 2 -rs 0 -mid 22 -tid 1 -ni 50 -li 50 -qo 4 -fd
```

Comment:
I'm looking at the output of this sample run and it is not clear if the mesh was untangled. Maybe print `min det(J)` after the optimization?

Reply:
Same in the parallel sample run.

Reply:
It prints with more verbosity eg `-vl 1`. Do you want to have that info with the `-vl 0`?

Reply:
For the untangling sample runs it makes sense to see that -- maybe just add `-vl 1` to those sample runs?

Reply:
Done.